.gitignore

```diff
@@ -8,7 +8,10 @@
 *.html
 .vscode
 .idea
-.DS_Store
+*.DS_Store
```

Comment:
There were many .DS_Store files being created on Mac on each of the subdirectories while working. To avoid adding them into the git commits we have it as wildcard.

Reply:
globs like ".DS_Store" which do not contain a "/" match any file in the repository with that name in any directory, see e.g. https://www.atlassian.com/git/tutorials/gitignore

Reply:
Yes, that is right. What would you suggest? changing the ignore pattern to `**/.DS_Store` maybe?